plugins/main/macvlan/macvlan_test.go

```diff
@@ -54,6 +55,7 @@ type Net struct {
 	DNS           types.DNS              `json:"dns"`
 	RawPrevResult map[string]interface{} `json:"prevResult,omitempty"`
 	PrevResult    types100.Result        `json:"-"`
+	LinkContNs    bool                   `json:"linkInContainer"`
```

Comment:
This field, `LinkContNs` does not have `omitempty`. It could be optional field, so we could add `omitempty` in json annotation.

It is minor case, only for re-encode this structure again by someone, but it might be good to annotate.

Reply:
Done